setup.py

```diff
@@ -43,7 +43,7 @@ def run(self):
                 if strict or missing:
                     log.warn('rebuilding js and css failed')
                     if missing:
-                        log.error('missing files: %s' % missing)
+                        log.error(f'missing files: {missing}')
```

Comment:
is this the only one to omit @jtpio ?

Reply:
This was mostly related to @maartenbreddels comment above: https://github.com/voila-dashboards/voila/pull/419#issuecomment-561089304

Thanks for this and the rebase!

Reply:
Looking at the diff, there seems to be a couple more.